docker/compute_worker/compute_worker.py

```diff
@@ -342,9 +357,12 @@ def _get_bundle(self, url, destination, cache=True):
         websocket = await websockets.connect(self.websocket_url)
         websocket_errors = (socket.gaierror, websockets.WebSocketException, websockets.ConnectionClosedError, ConnectionRefusedError)
 
-        while any(v["continue"] for v in logs.values()):
+        while any(v["continue"] for k, v in self.logs[kind].items() if k in ['stdout', 'stderr']):
             try:
-                for value in logs.values():
+                for key, value in self.logs[kind].items():
+                    # todo: just grab the keys we care about instead of "continuing" from keys we don't care about
```

Comment:
Is this meaning to do something like:

```python
logs = [self.logs[kind][key] for key in ('stdout', 'stderr')]
for value in logs:
```

hope that's helpful... if not should we resolve this todo in this PR or push it off?

Reply:
Yea, that was what I meant to go back and add. Completely missed this Todo. Good catch. Will add.